alf/environments/gym_wrappers.py

```diff
@@ -331,6 +333,110 @@ def transform_observation(self, observation):
         return obs
 
 
+@alf.configurable
+class RandomFrameCrop(BaseObservationWrapper):
```

Comment:
The name does not suggest that the `Random` is `Episodic`. Shall we make it more obvious?